static/js/click_handlers.js

```diff
@@ -564,18 +564,22 @@ export function initialize() {
         });
 
     function do_render_buddy_list_tooltip(elem, title_data) {
-        elem.tooltip({
-            template: render_buddy_list_tooltip(),
-            title: render_buddy_list_tooltip_content(title_data),
-            html: true,
-            trigger: "hover",
-            placement: "bottom",
-            animation: false,
-        });
-        elem.tooltip("show");
+        tippy(elem[0], {
+            // Quickly display and hide tooltips
+            // so that they don't stick and overlap
+            // each other.
+            delay: 0,
 
-        $(".tooltip").css("left", elem.pageX + "px");
-        $(".tooltip").css("top", elem.pageY + "px");
+            content: render_buddy_list_tooltip_content(title_data),
+            arrow: true,
+
+            // keeping placement `left` doesn't hide other users in
+            // the list.
+            placement: "left",
+
+            allowHTML: true,
+            showOnCreate: true,
+        });
```

Comment:
This seems to work well, except that if you have a really long name, in a narrow (almost mobile web size) window, the tooltip goes off the left of the screen.

Is there something we can do to make that not happen?

Reply:
Fixed it based on width.